### PR TITLE
Update Nextag.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

**Comment by ivysrono** [comment]
#8899 has removed Altmetric.com , why it show again.

File: src/chrome/content/rules/Nextag.xml [hunk 1]
```diff
@@ -1,50 +1,52 @@
...
+	Different http/https content:
+		www.nextag.com
+		www42.nextag.com
+		merchants.nextag.com
+		travel.nextag.com
+
+		www.nextag.com.au
+		www.nextag.de
+		www.nextag.fr
+		www.nextag.it
```
Comment (J0WI) [inline src/chrome/content/rules/Nextag.xml]: There is also ^ and a m. subdomain, but I could not reproduce "Different http/https content".

File: src/chrome/content/rules/Nextag.com.xml [hunk 2]
```diff
@@ -4,24 +4,30 @@
...
+	Redirect to www:
+		^nextag.com
+
 	Show empty page:
 		www.nextag.com
 		www42.nextag.com
 
+		m.nextag.ca
 		www.nextag.com.au
 		www.nextag.fr
```
Comment (J0WI) [inline src/chrome/content/rules/Nextag.com.xml]: I still have no empty page for them.

Comment (ivysrono) [inline src/chrome/content/rules/Nextag.com.xml]: I get different results from ISP and Tor each time.
I'm not so sure now.

File: src/chrome/content/rules/Nextag.com.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<!--
+	403:
+		merchants.nextag.com
+		travel.nextag.com
```
Comment (J0WI) [inline src/chrome/content/rules/Nextag.com.xml]: Secure connection failed for me

File: src/chrome/content/rules/Nextag.com.xml [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<!--
+	403:
+		merchants.nextag.com
```
Comment (J0WI) [inline src/chrome/content/rules/Nextag.com.xml]: WFM

File: src/chrome/content/rules/Nextag.com.xml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+		travel.nextag.com
+
+		www.nextag.de
+		www.nextag.fr
```
Comment (J0WI) [inline src/chrome/content/rules/Nextag.com.xml]: both WFM

**Pull request merged**